### PR TITLE
Add IbisDoc for some child elements

### PR DESCRIPTION
**Pull request by gvanbrakel**

### CODE REVIEW

File: core/src/main/java/nl/nn/adapterframework/batch/IRecordHandlerManager.java [hunk 1]
```diff
@@ -38,6 +39,7 @@
 	/**
 	 * @param flow New flow to be added to the managed flow elements
 	 */
+	@IbisDoc("flow to be added to the managed flow elements")
```
Comment (mhdirkse) [inline core/src/main/java/nl/nn/adapterframework/batch/IRecordHandlerManager.java:42]: This description starts with a lower-caps letter. How about starting with a capital to form a sentence?

Comment (mhdirkse) [reply core/src/main/java/nl/nn/adapterframework/batch/IRecordHandlerManager.java:42]: I do not see a number here, so no order is specified here yet for the XSD. Do you want that the XSD requests the same order here as the order of the methods?

File: core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java [hunk 2]
```diff
@@ -219,9 +208,7 @@ public void registerChild(IRecordHandlerManager manager) throws Exception {
 		ConfigurationWarnings.add(this, log, "configuration using element 'child' is deprecated. Please use element 'manager'", ConfigurationWarnings.DEPRECATION_SUPPRESS_KEY, getAdapter());
 		registerManager(manager);
 	}
-	/**
-	 * Register a uniquely named manager.
-	 */
+	@IbisDoc({"1", "a uniquely named manager"})
```
Comment (mhdirkse) [inline core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java:211]: I do not agree with "1" for the order here. We already put some @IbisDoc annotations for sort orders in an earlier pull request. There we use 10, 20, 30 etc, such that we have the option to insert new methods anywhere in the existing sort order.

File: core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
-   Copyright 2013, 2018, 2020 Nationale-Nederlanden
+   Copyright 2013, 2018 Nationale-Nederlanden, WeAreFrank!
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java:2]: welk jaar?

**Review by nielsm5 — CHANGES_REQUESTED** [review]

**Review by nielsm5 — APPROVED** [review]

File: core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java (hunk 2 above)
Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/batch/StreamTransformerPipe.java]: done

**Pull request merged**